src/app/shared/error-message.html

```diff
@@ -0,0 +1,14 @@
+<h2 mat-dialog-title>Error Message</h2>
```

Comment:
Change this to a user-friendly message.
"OOPS Houston, we have a problem"
Or whatever. but error Message sound more like a debug line than a message title for the End User.

Reply:
That is very funny!!!! changed it!